sdcm/cluster.py

```diff
@@ -276,6 +276,7 @@ def __init__(self, name, parent_cluster, ssh_login_info=None, base_logdir=None,
         self._alert_manager: Optional[PrometheusAlertManagerListener] = None
 
         self.termination_event = threading.Event()
+        self._wait_for_stop_event = threading.Event()
```

Comment:
as we using this event only in wait_db_up to stop waiter. let's name it stop_wait_db_up_event

Reply:
fixed